ui/org.eclipse.pde.ui/src/org/eclipse/pde/internal/ui/pderesources.properties

```
@@ -1510,6 +1510,7 @@ PluginWorkingSet_deselectAll_label=Dese&lect All
 PluginDevelopmentPage_presentation=Plug-in Manifest Editor Presentation
 PluginGeneralInfoSection_lazyStart=Activate this plug-in when one of its classes is loaded
 PluginGeneralInfoSection_singleton=This plug-in is a singleton
+PluginGeneralInfoSection_bundleshape=Shape after P2 installation:
```

In general we could re-think if this should be made more neutral, but this can be done in a follow-up.
```suggestion
PluginGeneralInfoSection_bundleshape=Shape after installation:
```